src/caf/space/__main__.py

```diff
@@ -1,25 +1,36 @@
 """Main module."""
+import os
+from pathlib import Path
 import caf.space
+from caf.toolkit import LogHelper, ToolDetails
+
 
 if __name__ == "__main__":
     args = caf.space.inputs.SpaceArguments.parse()
     args.validate()
-
-    if args.mode == "GUI":
-        caf.space.SpaceUI()
-    elif not args.config_path:
-        raise ValueError(f"For a {args.mode} translation a config is required.")
+    out_path = args.out_path
+    ver = caf.space.__version__
+    details = ToolDetails("caf.space", ver[:5])
+    if out_path:
+        log_out = out_path
     else:
-        config = caf.space.ZoningTranslationInputs.load_yaml(args.config_path)
-        trans = caf.space.ZoneTranslation(config)
-        if args.mode == "spatial":
-            out = trans.spatial_translation().to_csv(
-                args.out_path / f"{config.zone_1.name}_{config.zone_2.name}_spatial.csv",
-                ignore_index=True,
-            )
+        log_out = Path(os.getcwd())
+    with LogHelper("SPACE", details, log_file=log_out / "SPACE.log"):
```

Comment:
Use `__package__` for the root logger (which will be 'caf.space').

```suggestion
    with LogHelper(__package__, details, log_file=log_out / "SPACE.log"):
```